### PR TITLE
Removing all directories and files related to the old format

### PR DESCRIPTION
**Pull request by quintesse**

### CODE REVIEW

**Comment by quintesse** [comment]
@gastaldi what shall we do with the CODEOWNERS file? Just turn the access to `*`? Or do we remove it?

**Comment by gastaldi** [comment]
@quintesse in this case I believe the best is to remove it. No sense in keeping it since everything is in one big file now

**Comment by quintesse** [comment]
@gastaldi ok, but doesn't the CODEOWNERS file also give all of them the rights to review and approve PRs etc?

**Comment by gastaldi** [comment]
That's something that now will need to be removed from the PR process. I don't see how would `CODEOWNERS` work with a single file

**Comment by quintesse** [comment]
I know it won't work to only allow people to edit their own stuff, but I was wondering if it could still give some other advantages, like for example that all get notified to review the changes made by others. But if you say there's no use than we just remove it.

File: metadata.yaml [hunk 1]
```diff
@@ -1,350 +0,0 @@
-missions:
```
Comment (Ladicek) [inline metadata.yaml]: I thought this file would stay? Or is it gonna be converted to some other format as well?

Comment (quintesse) [reply metadata.yaml]: Yes, it stays, but as a JSON file now. That way everything is easier parseable by most types of clients.

Comment (Ladicek) [reply metadata.yaml]: Gotcha, thanks!

**Pull request merged**